.github/scripts/jira/update-jira-issue.js

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env node
```

Comment:
Thoughts on eventually moving the tag patsing shell script into js to simplify the workflow have a single location for the changeset stuff?

Reply:
yeah sounds good - I'll create a ticket